TV by the Numbers.js

```diff
@@ -1,144 +1,208 @@
 {
 	"translatorID": "180a62bf-efdd-4d38-8d85-8971af04dd85",
 	"label": "TV by the Numbers",
-	"creator": "odie5533",
-	"target": "^https?://tvbythenumbers\\.com/",
-	"minVersion": "1.0",
+	"creator": "Sonali Gupta",
+	"target": "^https?://tvbythenumbers\\.zap2it\\.com/",
+	"minVersion": "3.0",
 	"maxVersion": "",
 	"priority": 100,
 	"inRepository": true,
 	"translatorType": 4,
-	"browserSupport": "g",
-	"lastUpdated": "2015-06-10 10:48:25"
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2017-07-07 14:06:32"
 }
 
 /*
-	TV by the Numbers - translator for Zotero
-	Copyright (C) 2010 odie5533
+	***** BEGIN LICENSE BLOCK *****
 
-	This program is free software: you can redistribute it and/or modify
-	it under the terms of the GNU General Public License as published by
+	Copyright © 2017 Sonali Gupta
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
 	the Free Software Foundation, either version 3 of the License, or
 	(at your option) any later version.
 
-	This program is distributed in the hope that it will be useful,
+	Zotero is distributed in the hope that it will be useful,
 	but WITHOUT ANY WARRANTY; without even the implied warranty of
-	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-	GNU General Public License for more details.
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
 
-	You should have received a copy of the GNU General Public License
-	along with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
 
-/*
-	This translator supports saving a snapshot of a single post and saving
-	the citation of many posts at once without visiting each post. Thus, it does
-	not save a snapshot when multiple citations are to be saved.
+	***** END LICENSE BLOCK *****
 */
 
-
-PUB_TITLE = "TV by the Numbers";
-XPATH_TITLE = "//title";
-XPATH_PAGES = null;
-XPATH_DATE = "substring-after(substring-before(string(//p[@class='posted_on']),' by '), 'on ')";
-RE_DATE = /(.*)/;
-XPATH_AUTHORS = "substring-after(string(//p[@class='posted_on']),' by ')";
-RE_AUTHORS = /(.*)/;
-
 function detectWeb(doc, url) {
-	/* site has lots of garbage, check we're on the right doc */
-	if (!xpath_string(doc, doc, XPATH_TITLE))
-		return;
-	var posts = doc.evaluate("count(//div[@class='post-alt blog'])", doc, null,
-		XPathResult.NUMBER_TYPE, null).numberValue;
-	if (posts  == 1)
-		return "webpage";
-	else if (posts > 1)
+	if (url.indexOf('/?s=') > -1 && getSearchResults(doc, true))
 		return "multiple";
+	else if (doc.body.className.indexOf("single-post") > -1)
+		return "webpage";
```

Comment:
Should we change this to `blogPost`, which might be less generic? Moreover, the website uses WordPress...